templates/layouts/roq-default/main.html

```diff
@@ -27,6 +27,15 @@
   .search-button:hover i {
     transform: scale(1.1);
   }
+  .search-result-document-title {
+    color: #263959;
+    font-size: 11pt;
+  }
+  .search-result-document-hit a {
+    color: #515151;
+    text-decoration: none;
+    font-size: 11pt;
+  }
```

Comment:
```suggestion
  .search-result-document-title {
    color: #263959;
    font-size: 0.9rem;
  }
  .search-result-document-hit a {
    color: #515151;
    text-decoration: none;
    font-size: 0.9rem;
  }
```

We usually avoid using pt unit. I would use rem instead, which is relative to the root.